test/unitTests/testAssets/TestZip.ts

```diff
@@ -4,30 +4,28 @@
  *--------------------------------------------------------------------------------------------*/
 
 import * as archiver from 'archiver';
+import { TestFile } from './TestFile';
 
-export const Files = [
-    {
-        content: "File 1",
-        path: "file1.txt",
+export class TestZip {
+    constructor(private readonly _buffer: Buffer, private readonly _files : TestFile[]){
+    }
 
-    },
-    {
-        content: "File 2",
-        path: "folder/file2.txt"
+    get buffer(): Buffer{
+        return this._buffer;
     }
-];
 
-export const Binaries = [
-    {
-        content: "Binary 1",
-        path: "binary1.txt"
-    },
-];
+    get size(): number{
+        return this._buffer.length;
+    }
 
+    get files(): TestFile[]{
+        return this._files;
+    }
+}
 
-export async function createTestZipAsync(filesToAdd: Array<{ content: string, path: string }>): Promise<Buffer> {
+export async function createTestZipAsync(...filesToAdd: TestFile[]): Promise<TestZip> {
```

Comment:
I think you could make this a static method on TestZip and avoid having to import TestZip and this method in every file

Reply:
Done. Thanks!